src/translations.js

```diff
@@ -11,6 +11,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       it: `Statistiche GitHub di ${encodeHTML(name)}`,
       ja: `${encodeHTML(name)}のGitHub統計`,
       kr: `${encodeHTML(name)}의 GitHub 통계`,
+      nl: `${encodeHTML(name)}'${apostrophe} GitHub Statistieken`,
       "pt-br": `Estatísticas do GitHub de ${encodeHTML(name)}`,
```

Comment:
na de apo geen s?

Reply:
${apostrophe} zet wel of geen S achter de apo, gebaseerd op of de laatste letter X of S is.

Reply:
oh nice, wist ik niet.